### PR TITLE
Bsv20 - v2

### PR DESCRIPTION
**Pull request by shruggr**

### CODE REVIEW

File: bsv20.md [hunk 1]
```diff
@@ -10,34 +10,61 @@ The purpose of this proposal is to provide a standard that offers the same funct
...
+| op | Yes | Operation: `mint` |
+| amt | Yes | Supply of token. Max 2^64-1 |
+| dec | No | Decimals: set decimal precision, defaults to 0. This is different from BRC20 which defaults to 18 |
+
+
+*Example*:
+To mint a token, you would create an inscription with the following json (ContentType: application/bsv-20):
+```
+{ 
+  "p": "op-20",
```
Comment (jadwahab) [inline bsv20.md:39]: typo "op-20"

File: bsv20.md [hunk 2]
```diff
@@ -10,34 +10,61 @@ The purpose of this proposal is to provide a standard that offers the same funct
...
+  "op": "mint",
```
Comment (jadwahab) [inline bsv20.md:40]: will there be no deploy operation for the tickerless deploy+mint?

File: bsv20.md [hunk 3]
```diff
@@ -77,43 +104,50 @@ To mint `ordi` tokens, you would create an inscription with the following json (
...
-2- Transferring part of an inscription balance (for example, if you've minted 1000 `ordi` tokens and you want to send only 200 of those tokens to someone)
-
-
-Let's say you've minted 1000 `bank` tokens 3 times (as seen in image below). Your options are to either send one of the whole 1000 tokens to someone (option 1 below) or split that up and send send part of the 1000 tokens, let's say 200 tokens, to someone (option 2 below).
-
-<figure><img src="unisat-example-brc20.png" alt=""><figcaption></figcaption></figure>
+Using the same procedure as regular Satoshi transfers allows us to benefit from the parallelisation that regular Satoshis in Bitcoin benefit from where you can split a specific UTXO with a large amount into smaller UTXOs and spend those in parallel (the same way you could exchange a $100 bill into $1 bills and spend those in parallel) with no sequential bottlenecks that something like ERC20 sufffers from.
+| Key | Required? | Description |
+| --- | --------- | ----------- |
+| p | Yes | Protocol: `op-20` |
```
Comment (jadwahab) [inline bsv20.md:116]: typo `op-20`

File: bsv20.md [hunk 4]
```diff
@@ -77,43 +104,50 @@ To mint `ordi` tokens, you would create an inscription with the following json (
...
+**Inputs**
+1. previous transfer output holding 500 tokens
+2. previous transfer output holding 500 tokens
 
-In order to transfer a subset/part of the inscription balance (for example, sending only 400 of the minted or received 1000 tokens), you must follow the procedure described below:  
+**Outputs**
 
-Using the same procedure as regular Satoshi transfers allows us to benefit from the parallelisation that regular Satoshis in Bitcoin benefit from where you can split a specific UTXO with a large amount into smaller UTXOs and spend those in parallel (the same way you could exchange a $100 bill into $1 bills and spend those in parallel) with no sequential bottlenecks that something like ERC20 sufffers from.
+P2PKH inscription
+- {"p":"op-20","op":"transfer","id":"3b313338fa0555aebeaf91d8db1ffebd74773c67c8ad5181ff3d3f51e21e0000_1","amt":"100"}
```
Comment (jadwahab) [inline bsv20.md:140]: typo

File: bsv20.md [hunk 5]
```diff
@@ -77,43 +104,50 @@ To mint `ordi` tokens, you would create an inscription with the following json (
...
 
-In order to transfer a subset/part of the inscription balance (for example, sending only 400 of the minted or received 1000 tokens), you must follow the procedure described below:  
+**Outputs**
 
-Using the same procedure as regular Satoshi transfers allows us to benefit from the parallelisation that regular Satoshis in Bitcoin benefit from where you can split a specific UTXO with a large amount into smaller UTXOs and spend those in parallel (the same way you could exchange a $100 bill into $1 bills and spend those in parallel) with no sequential bottlenecks that something like ERC20 sufffers from.
+P2PKH inscription
+- {"p":"op-20","op":"transfer","id":"3b313338fa0555aebeaf91d8db1ffebd74773c67c8ad5181ff3d3f51e21e0000_1","amt":"100"} 
+  
+P2PKH inscription
+- {"p":"op-20","op":"transfer","id":"3b313338fa0555aebeaf91d8db1ffebd74773c67c8ad5181ff3d3f51e21e0000_1","amt":"500"}
```
Comment (jadwahab) [inline bsv20.md:143]: typo

File: bsv20.md [hunk 6]
```diff
@@ -77,43 +104,50 @@ To mint `ordi` tokens, you would create an inscription with the following json (
...
-Please note that with BSV-20 (unlike with BRC-20 on BTC) the process is only 1 step not 2 steps - you don't need to move tokens to a "transferrable" balance. This provides a better user experience and avoids issues associated with indexing (especially at scale).
 
-| Key  | Required? | Description                                                      |
-| ---- | --------- | ---------------------------------------------------------------- |
-| p    | Yes       | Protocol: Helps other systems identify and process bsv-20 events |
-| op   | Yes       | Operation: Type of event (Deploy, Mint, Transfer)                |
-| tick | Yes       | Ticker: 4 letter identifier of the bsv-20                        |
-| amt  | Yes       | Amount to transfer: States the amount of the bsv-20 to transfer. |
+P2PKH inscription
+- {"p":"op-20","op":"transfer","id":"3b313338fa0555aebeaf91d8db1ffebd74773c67c8ad5181ff3d3f51e21e0000_1","amt":"400"}
```
Comment (jadwahab) [inline bsv20.md:147]: typo -  just find and replace all

**Review by jadwahab — CHANGES_REQUESTED** [review]

**Review by rohenaz — APPROVED** [review]
lgtm

**Pull request merged**